addons/web/static/tests/actions/helpers.js

```diff
@@ -64,27 +66,29 @@ export async function createWebClient(params) {
     wc.destroy();
   });
   wc._____testname = QUnit.config.current.testName;
-  addLegacyMockEnvironment(wc, params.testConfig, params.legacyParams);
   await legacyExtraNextTick();
   return wc;
 }
 
 /**
  * Remove this as soon as we drop the legacy support
  */
-function addLegacyMockEnvironment(comp, testConfig, legacyParams = {}) {
+function addLegacyMockEnvironment(env, testConfig, legacyParams = {}) {
+  if (env instanceof owl.Component) {
```

Comment:
we have a single usecase, and you change it to pass the env instead of a component, so no need to handle both situations